namespace/id.go

```diff
@@ -4,22 +4,27 @@ import "bytes"
 
 type ID []byte
 
+// Less returns true if nid < ID, otherwise, false.
```

Comment:
[optional] since both types that are being compared are of type `ID` I think the variable name helps disambiguate

```suggestion
// Less returns true if nid < other, otherwise, false.
```

Reply:
Addressed https://github.com/celestiaorg/nmt/pull/96/commits/26b1ea676ac264bb2c6894efcd7a4f4aaf3924b3